src/views/components/shared/track/track.js

```diff
@@ -107,7 +121,7 @@ class Track extends Component {
           menu.append(menuItemToRemoveTrack);
         }
         else {
-          menu.insert(0, menuItemToAddTrack);
+          menu.insert(++position, menuItemToAddTrack);
```

Comment:
can you explain a bit why you use `position` ? XD?

Reply:
Maintains a certain hierarchy in context menu..? :sweat_smile: 
Anyways, i tried inserting at position 0 like before and it breaks without an error.